test/src/test/java/org/corfudb/integration/CorfuReplicationLargeTxIT.java

```diff
@@ -44,7 +44,7 @@ public class CorfuReplicationLargeTxIT extends LogReplicationAbstractIT {
         Sample.Metadata>> mapNameToMapStandby = new HashMap<>();
 
     private static final int NUM_ENTRIES_PER_TABLE = 20;
-    private static final int MAX_WRITE_SIZE = 7000;
+    private static final int MAX_WRITE_SIZE = 8000;
```

Comment:
Just curious, why the original PR with this change passed the integration test? https://github.com/CorfuDB/CorfuDB/pull/3501/checks

Reply:
It seems like the schema option change alone on master didn't exceed the threshold, but together with the changes in lr_v2 branch the serialized size exceeded the limit...